package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "business-create-ui",
-      "version": "5.5.0",
+      "version": "5.5.1",
```

Comment:
If you use `npm i` after updating the package.json file, this file gets updated automatically.